getusermedia.html

```diff
@@ -5513,13 +5517,14 @@ <h1>Privacy and Security Considerations</h1>
     <a href="https://w3ctag.github.io/design-principles/#device-enumeration">
     device enumeration design principles</a>.
     </p>
-    <p>For origins to which permission has been granted, the
-    <a data-link-type=event>devicechange</a> event will be emitted across browsing
+    <p>For web documents where capture has taken place or for web documents in focus, the
```

Comment:
> For web documents where capture has taken place

Do you mean within the current browsing session?

Reply:
Probably, except "browsing session" isn't a well-established term either in the spec or in layperson terms I fear.

Technically it's [`Document`](https://html.spec.whatwg.org/multipage/dom.html#document), but we seem to want to be less technical in these sections, so I went with "web document", but perhaps "open web document" is better?

```suggestion
    <p>For open web documents where capture has taken place, or for open web documents that have focus, the
```